8748 - made the glossary button a component

brianpetway opened the pull request:
https://federal-spending-transparency.atlassian.net/browse/DA-8748

anjenkin commented on src/components/glossary/glossary-button/glossary-button.jsx:
```diff
@@ -0,0 +1,16 @@
+export default function GlossaryButton() {
+  return(
+    <div>
+      <button id="afg-floating-glossary-button" tabIndex={-1}>
```
I think we should try to separate out the CSS that specific to this component into it's own css module.

anjenkin commented on src/components/glossary/glossary.jsx:
```diff
@@ -457,12 +456,7 @@ export default function Glossary() {
-						Glossary
-					</div>
-				</button>
+				<GlossaryButton />
```
This button will have to be outside of the Glossary component when we hide the Glossary on the initial page load. 
 Otherwise it will only be visible after opening the Glossary.  We could add it to the header component though it feels like it would make more sense in the footer.  I think what ever way requires the least refactoring of css makes sense.

anjenkin reviewed (changes requested):
See feedback inline

brianpetway commented:
Moved the button to the page header, where the glossary component is; moved the css for the button to the glossary-button dir; the button is functional at this time, but it's still using the d3.select in the glossary.jsx for its click function
